modules/extensions/package.json

```diff
@@ -32,15 +32,15 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
-    "@luma.gl/shadertools": "9.0.0-beta.1"
+    "@luma.gl/shadertools": "9.0.0-beta.3"
   },
   "peerDependencies": {
     "@deck.gl/core": "^9.0.0-alpha",
-    "@luma.gl/constants": "9.0.0-beta.1",
-    "@luma.gl/core": "9.0.0-beta.1",
-    "@luma.gl/engine": "9.0.0-beta.1",
-    "@luma.gl/shadertools": "9.0.0-beta.1",
-    "@luma.gl/webgl": "9.0.0-beta.1",
+    "@luma.gl/constants": "9.0.0-beta.3",
+    "@luma.gl/core": "9.0.0-beta.3",
+    "@luma.gl/engine": "9.0.0-beta.3",
+    "@luma.gl/shadertools": "9.0.0-beta.3",
+    "@luma.gl/webgl": "9.0.0-beta.3",
```

Comment:
This should no longer be needed?

Reply:
Extensions is still using WEBGLRenderBuffer. I have asked @felixpalmer if he can update CollisionFilterExtension to use a depth texture instead.

Reply:
https://github.com/visgl/deck.gl/pull/8477